quill-engine/src/main/scala/io/getquill/ast/Ast.scala

```diff
@@ -422,7 +425,7 @@ object Visibility extends OpinionValues[Visibility] {
 }
 
 sealed trait Renameable extends Opinion[Renameable] {
-  def fixedOr[T](plain: T)(otherwise: T) =
+  def fixedOr[T](plain: => T)(otherwise: => T): T =
```

Comment:
This wasn't lazy enough. Always computing both sides even if one of them is never used